@Test
public void testSyncAllExternalUsersThrowingHandler() throws Exception {
    String[] result = createThrowingSyncMBean(false).syncAllExternalUsers();
    Map<String, String> expected = getExpectedUserResult("ERR", false);
    assertResultMessages(result, expected);
}
